homework.rb

```diff
@@ -1,6 +1,7 @@
-VARIANT = 0
-
+VARIANT = 1
+# "1234-5678-1234"
 def generate
   # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  a, b, c = rand(1000...9999), rand(1000...9999), rand(1000...9999)
```

Comment:
Also you 3 times called `rand(1000...9999)`. What about some DRY (don't repeat yourself) ?